### ISSUE TITLE
Investigate/ Implement Neo4j Metrics & Health

### ISSUE THREAD

**Issue by noheton** [noheton/shepard#276]
***In GitLab by @xit-max-hey on Oct 22, 2024, 11:10 GMT+2:***

# Research Task

## :goal_net: Goal

After implemeting Neo4j the "Quarkus Way", we can try out some features that come with Quarkus like the "Connection Health" or the "Driver Metrics" (see this [link](https://docs.quarkiverse.io/quarkus-neo4j/dev/index.html#_connection_health_check) for more information).

We should investigate if these features, especially the metrics provide a benefit to our application.
One idea could be to use the metrics to find and investigate performance issues with Neo4j queries.

## :thinking_face: Risks & Open Questions

*Migrated from GitLab: https://gitlab.com/dlr-shepard/shepard/-/work_items/276*

**Comment by noheton** [comment]
***In GitLab by @xit-max-hey on Oct 22, 2024, 13:27 GMT+2:***

marked this issue as blocked by #274

**Comment by noheton** [comment]
***In GitLab by @xit-paul-reinhardt on Feb 21, 2025, 10:11 GMT+1:***

marked this issue as a duplicate of #274
